### PR TITLE
HDDS-7102. Exclude unnecessary RangerClient dependencies

### PR DESCRIPTION
**Pull request by smengcl**
## What changes were proposed in this pull request?

Exclude:

```
com.amazonaws:aws-java-sdk-bundle
org.apache.hive:hive-storage-api
org.apache.kafka:kafka-clients
org.apache.solr:solr-solrj
```

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-7102

## How was this patch tested?

- [x] Check new licenses, if any.
- [x] All existing tests should pass.
- [ ] Ensure `RangerClient` would still function as expected.

### CODE REVIEW

**Review by adoroszlai — CHANGES_REQUESTED** [review]
Thanks @smengcl for working on this.  Tarball size is reduced to 305M, which looks much better.

File: hadoop-ozone/dist/src/main/license/bin/LICENSE.txt [hunk 1]
```diff
@@ -438,6 +442,13 @@ angular-nvd3-1.0.9.min.js
 angular-1.8.0.min.js
 jquery-3.5.1.min.js
 
+
+LGPL 2.1
+=====================
+
+   net.java.dev.jna:jna
+   net.java.dev.jna:jna-platform
```
[the remark this answers is not in the document]
Comment (smengcl) [reply hadoop-ozone/dist/src/main/license/bin/LICENSE.txt:450]: Ah. I actually misread its LICENSE. I thought it was LGPL only after 2.1.

Done

**Review by adoroszlai — APPROVED** [review]
Thanks @smengcl for updating the patch.

**Comment by smengcl** [comment]
Thanks @adoroszlai for reviewing and merging this.

**Pull request merged**